### ISSUE TITLE
Missing/Broken skip link

### ISSUE THREAD

**Issue by camillevilla** [sul-dlss/sul-bento-app#287]
There are no skip links at the top of the page, but there’s a vestigial link inside the `<main>` section. 

```
<div class="skip-link">
<a href="#main-content"></a>
</div>
```

Delete this div
Add skip links to Search and Main content, following the SearchWorks patter
